drainpipe-dev/composer.json

```diff
@@ -10,7 +10,7 @@
         "psr-4": {"Lullabot\\DrainpipeDev\\Tests\\Functional\\": "tests/src/functional/"}
     },
     "require": {
-        "php": "^8.1",
+        "php": "^8.3.8",
```

Comment:
We're going to want to support PHP 8.1, 8.2, and 8.3.

Reply:
I'm unclear about what to do next.
If I don't change the PHP version, it won't work.
Do you have a suggestion on how to do it? My approach may be wrong.